### PR TITLE
Add configurable-http-proxy to the recipe

### LINKED ISSUE #9: add configurable-http-proxy to recipe
From Riot discussion. I think I forgot to include it, and it worked only because I had installed it globally before.

### PR DESCRIPTION
**Pull request by kinow**
Closes #9

### CODE REVIEW

**Review by matthewrmshin — COMMENTED** [review]
Looks good. I'll try it out later in a clean conda environment.

File: recipes/cylc/meta.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ source:
 build:
   # Skip as Cylc requires Linux
   skip: True  # [not linux]
-  number: 0
+  number: 1
```
Comment (kinow) [inline recipes/cylc/meta.yaml:22]: See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string for more:

>The build number should be incremented for new builds of the same version. The number defaults to 0. The build string cannot contain "-". The string defaults to the default conda build string plus the build number.

**Comment by kinow** [comment]
>Looks good. I'll try it out later in a clean conda environment.

Thanks @matthewrmshin. I did not upload this version, even though I increased the build number. So `cylc install conda` will download the old version/build. Only way to test it now is as commented in the issue linked. Via `conda build .` && the `conda install --use-local` I think.

File: recipes/cylc/meta.yaml (hunk 1 above)
Comment (hjoliver) [reply recipes/cylc/meta.yaml:22]: Good to know.

**Review by matthewrmshin — APPROVED** [review]

**Review by hjoliver — APPROVED** [review]

**Pull request merged**